carta/session.py

```diff
@@ -417,6 +417,43 @@ def clear_raster_scaling_reference(self):
         """Clear the raster scaling reference."""
         self.call_action("clearRasterScalingReference")
 
+    # VIEWER MODES
+    @validate(Boolean())
+    def set_viewer_mode(self, multiple=True):
+        """
+        Switch between single-panel mode and multiple-panel mode.
+
+        Parameters
+        ----------
+        multiple : {0}
+            Whether to use multiple-panel mode. By default is ``True``.
```

Comment:
The default sentence should either be `Default: something.`, `The default is something.`, or `Defaults to something.` Sometimes we do use `By default [full sentence here...]`, but "by" should not be used here. (This also applies to the other description.)

Reply:
Done in d44b668.